### ISSUE TITLE
New release to support pytest>=5.4.0

### ISSUE THREAD

**Issue by simonfagerholm** [pytest-dev/pytest-asyncio#150]
@Tinche I'm not sure what the release process looks like, but I think it would be time for a new release soon.
Sorry to bother you all the time, I image you might be quite busy but I not sure who else is able to approve changes in the repo.

I imagine the following is needed:
* Write the changelog in README.rst
* Bump the version in `__init__.py` to 0.11.0
* Make release build and publish to pypi
* Bump the version in `__init__.py` to 0.12.0.dev

**Comment by Tinche** [comment]
I'm waiting in another PR to rebase (https://github.com/pytest-dev/pytest-asyncio/pull/133), then we can cut a release 👍

**Comment by simonfagerholm** [comment]
Awesome! I missed that one

**Comment by fulaphex** [comment]
@Tinche any progress on that?

**Comment by simonfagerholm** [comment]
@fulaphex Still waiting on @asvetlov to get the fixes from master in #133

**Comment by Tinche** [comment]
New release probably going out today!

**Comment by fulaphex** [comment]
@Tinche is there any update on the release? is there anything else that needs to be done before the release?

**Comment by Tinche** [comment]
Whoops, this slipped my mind. Today!

**Comment by Tinche** [comment]
The release is out!
